.github/workflows/emscripten.yml

```diff
@@ -67,7 +67,7 @@ jobs:
         with:
           persist-credentials: false
 
-      - uses: pypa/cibuildwheel@d04cacbc9866d432033b1d09142936e6a0e2121a # v2.23.2
+      - uses: pypa/cibuildwheel@e91560e8a14470ff35488e414aa7aa1de5aec2d8 # v2.23.2
```

Comment:
this is now painful cause the comment is not synced with the hash

Reply:
Looks like a dependabot bug to me ... dependabot is supposed to update the version comment as well ...